src/main/res/values/strings.xml

```diff
@@ -749,33 +749,33 @@
 
     <!-- 2016-04-03 -->
     <string name="error_tor_not_installed">Tor will not work unless Orbot is installed. Install now?</string>
-    <string name="error_tor_setting_failed">Failed to set proxy.</string>
+    <string name="error_tor_setting_failed">Could not set proxy.</string>
 
     <!-- 2016-04-10 -->
     <string name="error_parse_imgur_title">Parse Error</string>
-    <string name="error_parse_imgur_message">Imgur sent some data I couldn\'t understand.</string>
+    <string name="error_parse_imgur_message">Imgur sent incomprehensible data.</string>
```

Comment:
Preferred the original. Imgur might not be at fault here :p

Reply:
Who is "I", is the bigger question I am left with.
Arguably comprehension is on part of RedReader.

Reply:
The original was fun and quirky, whereas the new one is terse, frigid, and arguably accusatory. Let's just leave it as it was :smile:

Reply:
The software (hopefully) isn't sentient.